web/admin/src/App.js

```diff
@@ -246,6 +253,31 @@ export default class App extends Component {
     )
   }
 
+  parseDataForExport = () => {
```

Comment:
Parsing means turning text into structured data, which is not what is happening in this function.